config.yaml

```
@@ -42,17 +42,20 @@ options:
       Amount of disk space to allocate to root disk for virtual machine runner. Positive integers
       with KiB, MiB, GiB, TiB, PiB, EiB suffix.
   reconcile-interval:
-    type: int
-    default: 10
+    type: float
+    default: 10.0
     description: >
       Minutes between each reconciliation between the current state and the target state of the
       runners. On reconciliation the charm polls the state of runners, such as, the number of
-      container runners, and see if actions are needed.
+      container runners, and see if actions are needed. A jitter of 25% is added. Changes to this
+      time interval will apply when next interval is triggered.
   update-interval:
-    type: int
-    default: 60
+    type: float
+    default: 60.0
     description: >
-      Minutes between each check for new versions of the runner binary.
+      Minutes between each check for new versions of the runner binary and services the charm
+      depends on. A jitter of 25% is added. Changes to the time interval will apply when next
```

```suggestion
depends on. A random delay of 25% is added. Changes to the time interval will apply when next
```
same suggestion as above :)